Game/lv/Game.po

```diff
@@ -2524,84 +2524,84 @@ msgstr "Mēs sazinājāmies ar Arque Korporāciju, taču viņi noliedza, norādo
 msgid "The Department of Maritime Traffic Investigation declined to comment, after stating in last weeks' issue that they were \"confident that this won't impact any passing traffic\"."
-msgstr ""
+msgstr "Jūras Satiksmes Izmeklēšanas Departaments atteicās sniegt komentārus pēc tam, kad pagājušās nedēļas numurā bija paziņojis, ka \"ir pārliecināts, ka tas neietekmēs garāmbraucošo satiksmi\"."
 
 #. Key:	Paragraph11
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Paragraph11"
 msgid "The colonies of algae situated on the island remain strong, as recent surveys show a vast increase in the amount of so called \"red algae\", the island's very own unique species of algae continues to take over lakes, and other natural bodies of water."
-msgstr ""
+msgstr "Uz salas esošās aļģu kolonijas joprojām ir spēcīgas, jo nesen veiktie pētījumi liecina par ievērojamu tā saukto \"sarkano aļģu\" daudzuma pieaugumu, salas unikālā aļģu suga turpina pārņemt ezerus un citas dabiskas ūdenstilpes."
 
 #. Key:	Paragraph12
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 #: /Game/Resources/Localisation/ST_Newspaper01.ST_Newspaper01
 msgctxt "ST_Newspaper01,Paragraph12"
 msgid "Local resident Rosie Hardman considers the algae a menace, and would be happier if it were gone, saying \"It [the algae] is taking over, we need to get rid of it as soon as possible. I don't want all of the water on the island to be stagnant. What is it good for anyway?\""
-msgstr ""
+msgstr "Vietējā iedzīvotāja Rozija Hārdmena aļģes uzskata par draudu un būtu priecīgāka, ja to vairs nebūtu, sakot: \"Tās [aļģes] pārņem visu, mums no tām jāatbrīvojas pēc iespējas ātrāk. Es nevēlos, lai viss salas ūdens ir nostāvējies. Kam tas vispār der?\""
 
 #. Key:	Body01
 #. SourceLocation:	/Game/Resources/Localisation/ST_Newspaper02.ST_Newspaper02
 #: /Game/Resources/Localisation/ST_Newspaper02.ST_Newspaper02
 msgctxt "ST_Newspaper02,Body01"
 msgid "The lighthouse situated in Redton on the land outjut is set to be decommissioned by the local mayor John Menn, following an investigation carried out by the coastguard.\r\n\r\n\"We're confident that this won't impact any passing traffic\" commented Bill Hayak of the Department of Maritime Traffic Investigation. \"Whilst at one time you could see cargo ships just off the coast of Redton, now it's no longer the case with shipping routes from the mainland being altered\" he added."
-msgstr ""
+msgstr "Pēc krasta apsardzes veiktās izmeklēšanas vietējais mērs Džons Menns plāno pamest bāku, kas atrodas Redtonā uz sauszemes.\r\n\r\n\"Esam pārliecināti, ka tas neietekmēs garāmbraucošo satiksmi,\" komentēja Jūras satiksmes izmeklēšanas departamenta pārstāvis Bils Hajaks. \"Lai gan kādreiz pie Redtonas krastiem varēja redzēt kravas kuģus, tagad kuģu ceļi no cietzemes vairs netiek mainīti,\" viņš piebilda."
```

Comment:
When I run this through Google Translate, I get:
> "While cargo ships used to be seen off the coast of Redton, shipping lanes from the mainland are no longer being altered," he added.

If that's anywhere near correct, the original meaning gets lost a bit - the speaker is trying to convey that the lighthouse is no longer necessary, because the shipping traffic is far lower than it once was.

A minor detail though